ui/src/pages/Devices.tsx

```diff
@@ -43,6 +43,15 @@ export const DeviceList = () => (
   <List>
     <Datagrid rowClick="show" bulkActionButtons={<DeviceListBulkActions />}>
       <TextField label="Hostname" source="hostname" />
+      <TextField label="Allowed IPs" source="allowed_ips" />
+      <TextField label="Tunnel IP" source="tunnel_ip" />
+      <TextField label="Local IP" source="local_ip" />
+      <TextField label="Organization Prefix" source="organization_prefix" />
+      <TextField label="Reflexive IPv4" source="reflexive_ipv4" />
+      <TextField
+        label="Endpoint Local IPv4"
+        source="endpoint_local_address_ipv4"
+      />
```

Comment:
This looks like too much detail for the list of all devices. I would leave most of this for the detailed view for when you click on a device to see all of its details. That is `DeviceShow` down below.

Reply:
Going to push back on that a bit. I personally don't like having to drill down to get basic information. EC2 has 15 columns. I trimmed this down some. I took out allowed IPs but a routes column might make sense to go back in eventually.